README.md

```diff
@@ -317,43 +317,23 @@ that wish to use it as a module. A bare-minimum flake.nix would be as follows:
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    home-manager.url = "github:rycee/home-manager";
+    home-manager.url = "github:nix-community/home-manager";
   };
 
-  outputs = inputs: {
+  outputs = { home-manager, nixpkgs, ... }: {
     nixosConfigurations = {
-      hostname = let
+      container = nixpkgs.lib.nixosSystem {
         system = "x86_64-linux";
-        pkgs = inputs.nixpkgs.legacyPackages.${system};
-        inherit (inputs.nixpkgs) lib;
-
-        # Things in this set are passed to modules and accessible
-        # in the top-level arguments (e.g. `{ pkgs, lib, inputs, ... }:`).
-        specialArgs = {
-          inherit inputs;
-        };
-
-        hm-nixos-as-super = { config, ... }: {
-          # Submodules have merge semantics, making it possible to amend
-          # the `home-manager.users` submodule for additional functionality.
-          options.home-manager.users = lib.mkOption {
-            type = lib.types.attrsOf (lib.types.submoduleWith {
-              modules = [ ];
-              # Makes specialArgs available to Home Manager modules as well.
-              specialArgs = specialArgs // {
-                # Allow accessing the parent NixOS configuration.
-                super = config;
-              };
-            });
-          };
-        };
-
         modules = [
           ./configuration.nix
-          inputs.home.nixosModules.home-manager
-          hm-nixos-as-super
+          home-manager.nixosModules.home-manager
+          ({ pkgs, ... }: {
```

Comment:
```suggestion
          ({
```

I think that should work. If not, though, `pkgs` should  at least be removed.

Reply:
Yeah, initially I've added some packages to user environment, but ultimately decided to drop those. Will check if config as a set will work

Reply:
Yeah, sets are fine